Delcam.ProductInterface.PowerMILL/Classes/PMBoundary.cs

```diff
@@ -135,6 +135,24 @@ public void Smash(double smashTolerance)
             PowerMill.DoCommand(string.Format("EDIT BOUNDARY \"{0}\" SMASH {1}", Name, smashTolerance));
         }
 
+        /// <summary>
+        /// Inserts the specified Boundary into this Boundary.
+        /// </summary>
+        /// <param name="boundary">The boundary to insert into this Boundary.</param>
+        public void InsertBoundary(PMBoundary boundary)
+        {
+            PowerMill.DoCommand("EDIT BOUNDARY '" + Name + "' INSERT BOUNDARY '" + boundary.Name + "'");
```

Comment:
Is it worth testing that the boundary is not null and that it exists in PowerMill?